packages/synapse-react-client/src/lib/containers/home_page/featured-data/FacetPlotsCard.tsx

```diff
@@ -179,11 +189,13 @@ const FacetPlotsCard: React.FunctionComponent<FacetPlotsCardProps> = ({
                       </div>
                     )}
                   </SizeMe>
-                  <FacetPlotLegend
-                    labels={plotData?.labels}
-                    colors={plotData?.colors}
-                    isExpanded={false}
-                  />
+                  <div className="FacetPlotsCard__body__legend">
```

Comment:
I moved other code to use mui Box and sx for inline layout, but this one I left (because the rest of FacetPlotsCard uses this style today).